about.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The LOTR Lovers Conference</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <div>
```

Comment:
- [ ] You seem to have forgotten to include the topmost navbar which includes the social media icons. Kindly include it to meet all the design requirements.

**This is the section that is missing from your desktop design**

![image](https://user-images.githubusercontent.com/86318284/212123111-1a6b3435-3516-4bd3-ad03-1e743cb772b4.png)

- [ ] As this is a navigation bar, kindly ensure that you use the semantically correct tag for the section which is header -> nav -> your elements.